dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        await db.close()
```

Comment:
The `await db.close()` ensures that the session is properly closed after use. This is important for releasing database connections back to the pool. Ensure that any exceptions during session usage are handled appropriately in the calling context.